lib/bot.js

```diff
@@ -50,6 +55,16 @@ Bot.prototype.attachListeners = function() {
   this.client.addListener('error', function(error) {
     logger.error('Received error event from IRC', error);
   });
+
+  this.client.addListener('invite', function(channel, from, message) {
+    logger.debug('Received invite', channel, from, message);
+    if (this.opt.channels.indexOf(channel) > -1) {
```

Comment:
Would probably be better to do `if (this.channelMapping[channel]) {` here instead, note that you need to do `.bind(this)` on the listener callback then so that `this` is the Bot-instance and not the IRC client itself.